### DISCUSSION TITLE
Keep the new details pane but Win10 Explorer

### DISCUSSION THREAD

**Discussion by StackOverflow0x** [valinet/ExplorerPatcher#2228] [Ideas]
I was updating to the latest version of Explorer Patcher after I had partially installed the latest Windows 11 update and I saw it gave me the new details pane, which I like, but with the Windows 10 Explorer windows, which I like. But upon finishing the update and restarting, I couldn't see a way to keep the two features like I saw in that brief state.
Could it be possible to have the newer, better details pane along with the Windows 10 Explorer? I was teased by what was possible and I can't stand Windows 11's Explorer and it's a big part of why I'm running Explorer Patcher in the first place.

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

**Comment by StackOverflow0x** [comment]
Windows 11 Version 22H2 (22621.2361)
Explorer Patcher Version 22621.2361.58.1

My Explorer Patcher settings for File Explorer are:
✕ Register as shell extension
✓ Disable the Windows 11 context menu
✕ Always use legacy file transfer dialog
✕ Use classic drive groupings in This PC
  Control Interface: Windows 10 Ribbon
✓ Use immersive menus when displaying Windows 10 context menus
✕ Disable navigation bar
✕ Disable modern search bar
✕ Shrink address bar height
✕ Hide search bar completely

**Reply by pyrates999** [reply]
Thank you.  It will also help if you could post a screen shot of what you want.

**Reply by StackOverflow0x** [reply]
Okay. Here's what I'm looking at when I switch it to the Windows 11 Command Bar for Control Interface. The layout of the sidebar is different, specifically how it shows larger image previews than the default Windows 10 Details pane.
![image](https://github.com/valinet/ExplorerPatcher/assets/22903042/1e733389-a1d0-4547-a584-414e739883ad)
![image](https://github.com/valinet/ExplorerPatcher/assets/22903042/4f8d8768-66b0-4262-abb7-7c90870f4480)
![image](https://github.com/valinet/ExplorerPatcher/assets/22903042/abf1a9c3-f054-4c95-a8a1-be75c950ae9a)

**Comment by pyrates999** [comment]
Thank you.  I also noticed you're running a preview update: https://support.microsoft.com/en-au/topic/september-26-2023-kb5030310-os-build-22621-2361-preview-363ac1ae-6ea8-41b3-b3cc-22a2a5682faf

Uninstall it if you want to use explorer patcher and don't install preview updates in the future: https://github.com/valinet/ExplorerPatcher/wiki/Symbols

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

All preview updates are then shown as download and install and are never automatically installed if you have the above unticked.

Preview updates are also not fully tested by Microsoft and thus could have bugs.

You should only be installing preview updates if you are a developer or are OK with running pre-release versions of EP.
